### ISSUE TITLE
Suggest an starting point for navigating the tiles in the tiles description

### ISSUE THREAD

**Issue by joanma747** [opengeospatial/OGC-API-Tiles#36]
TileJSON includes a tile center:

>// OPTIONAL. Default: null.
// The first value is the longitude, the second is latitude (both in
// WGS:84 values), the third value is the zoom level as an integer.
// Longitude and latitude MUST be within the specified bounds.
// The zoom level MUST be between minzoom and maxzoom.
// Implementations can use this value to set the default location. If the
// value is null, implementations may use their own algorithm for
// determining a default location.
"center": [ -76.275329586789, 39.153492567373, 8 ]

Can we include this to have a nice way to start in the tile descriptions .
For TMS "zoom level" is selecting one of the TileMatrix ID.

#31

**Comment by jeffharrison** [comment]
Yes, I think it could be one starting point for navigating the tiles. However, I must admit I don't understand how this part would work... 

'For TMS "zoom level" is selecting one of the TileMatrix ID.'

**Comment by pomakis** [comment]
(This recommendation was originally posted as response to issue #31.)

I think it'd be better called "areaOfInterest" or "aoi", because it might not necessarily be the center.  If the client wants the actual center, that can be trivially computed from either the extent reported at the collections/{collectionId} endpoint or from the bounds of the tile-matrix set.  Also, it'd be better to report a bounding-box extent rather than just a point, to give client applications the ability to determine a good initial view window regardless of the size or aspect ratio of its viewport.  Finally, it sounds like this concept belongs in "OGC API-Common Part 2: Collections".  Wouldn't it be equally as applicable to map layers, coverages and feature collections?

So my recommendation would be to augment the Collection Information Schema in "OGC API-Common Part 2: Collections" to include a optional "areaOfInterest" or "aoi" field that has the same schema as "extent", and whose value, if present, must be within the bounds specified by "extent".  E.g.:

```
  "extent": {
    "spatial": {
      "bbox": [
        35.9028738,
        32.4168138,
        36.5747694,
        33.1424348
      ]
    }
  },

  "areaIOfInterest": {
    "spatial": {
      "bbox": [
        36.1234235,
        32.7345543,
        36.3543345,
        33.0045642
      ]
    }
  },
```

Note that using the same schema as "extent" allows the area of interest to indicate elevation and temporal dimensions as well.
